### PR TITLE
KEYCLOAK-14795: getCredentialTypeMetadata take one arg

### PR DESCRIPTION
**Pull request by M3lkior**

### CODE REVIEW

File: server_development/topics/auth-spi.adoc [hunk 1]
```diff
@@ -658,8 +658,7 @@ The config values can be defined within the admin console if you set up config d
     }
 ----
 
-Last method in SecretQuestionCredentialProvider class is getCredentialTypeMetadata(), which is an abstract method of CredentialProvider
-interface. Each Credential provider has to provide and implement this method. The method returns an instance of CredentialTypeMetadata,
+Last method in SecretQuestionCredentialProvider class is getCredentialTypeMetadata(CredentialTypeMetadataContext metadataContext), which is an abstract method of CredentialProvider interface. Each Credential provider has to provide and implement this method. The method returns an instance of CredentialTypeMetadata,
```
Comment (andymunro) [inline server_development/topics/auth-spi.adoc]: Please add "the" before "Last method" and "CredentialProvider interface."

**Review by andymunro — CHANGES_REQUESTED** [review]
Just a minor comment. Thanks.

File: server_development/topics/auth-spi.adoc (hunk 1 above)
Comment (M3lkior) [reply server_development/topics/auth-spi.adoc]: Done

**Comment by M3lkior** [comment]
Hi @andymunro , i do changes in the PR.

Best regards

File: server_development/topics/auth-spi.adoc [hunk 2]
```diff
@@ -658,7 +658,7 @@ The config values can be defined within the admin console if you set up config d
     }
 ----
 
-Last method in SecretQuestionCredentialProvider class is getCredentialTypeMetadata(CredentialTypeMetadataContext metadataContext), which is an abstract method of CredentialProvider interface. Each Credential provider has to provide and implement this method. The method returns an instance of CredentialTypeMetadata,
+The last method to implement in SecretQuestionCredentialProvider class is getCredentialTypeMetadata(CredentialTypeMetadataContext metadataContext), which is an abstract method of the CredentialProvider interface. Each Credential provider has to provide and implement this method. The method returns an instance of CredentialTypeMetadata,
```
Comment (andymunro) [inline server_development/topics/auth-spi.adoc]: Looks good except we need a "the" before "SecretQuestionCredentialProvider"

**Review by andymunro — CHANGES_REQUESTED** [review]
One minor changes. Thanks.

File: server_development/topics/auth-spi.adoc (hunk 2 above)
Comment (M3lkior) [reply server_development/topics/auth-spi.adoc]: it is done @andymunro

**Pull request merged**